components/styled/table/src/atoms/table.tsx

```diff
@@ -8,6 +8,18 @@ const Container = styled.div`
   background-color: white;
   border-radius: 6px;
   display: inline-block;
+  max-width: calc(100%);
```

Comment:
Think you can get away with just `max-width: 100%` here

Reply:
& on line 12

Reply:
calc(100%) will calculate the size of the container when screen is being resized without this scroll wont work. the actual width and height will be defined by the parent component using this element so the 100% will result in the width and height of the parent container whatever that is e.g. if its defined 800px width the table will max out to 800 and any overflow will be scrollable then.

Reply:
The styles will reflow whenever the DOM or viewport is changed, and the max-width property will work as you describe whenever that happens 😊. Pretty sure you don't need the `calc` bit.

`calc` is good when you're doing something like 100% - 10px and the css processor needs to perform some arithmetic calculation